tools/uniswapv2/detect.py

```diff
@@ -102,15 +101,15 @@ def pairs_and_token_details_from_graph() -> Dict[str, Any]:
         result = graph.query(querystr, param_types=param_types, param_values=param_values)
         for entry in result['pairs']:
             contracts.append({
-                'address': to_checksum_address(entry['id']),
+                'address': deserialize_ethereum_address(entry['id']),
```

Comment:
handle a potential `DeserializationError` here. No fancy way needed. Just a `print()` of the error followed by a `sys.exit(1)`.

This is just a script to detect uniswap lp pools.

Combine these 3 deserializations into one place and handle their errors together.